### DISCUSSION TITLE
Notification of losing compliance

### DISCUSSION THREAD

**Discussion by duglin** [cncf/clomonitor#659] [Q&A] [answered]
Is there a way to be notified if our repos suddenly become non-compliant? Either due to our stuff changing or due to new requirements being added to the CLO stuff.

**Comment by tegioz** [comment]
Hi @duglin

~~No, there isn't at the moment~~ (there is, actually, please see answer below). But eventually when checks are more stable we definitely would like to include some form of built-in alerts 🙂

Maybe embedding a report summary somewhere frequently viewed in the project's website or readme could help noticing changes earlier in the meantime. They are generated dynamically and refreshed very often, so they should always be up to date.

<img width="1381" alt="report-summary" src="https://user-images.githubusercontent.com/1213902/194344884-b8622c81-201f-4b11-afa0-b61cf46ca3e6.png">

**Comment by tegioz** [comment, marked as answer]
@duglin there is actually a way you could get notified right now: you can run the `clomonitor-linter` periodically (as a [GitHub scheduled workflow](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule), for example) with your desired pass score. The workflow will fail if the score is below the threshold provided (you can set it to 100 if you wish), which would trigger the notification.

You can find a similar example [here](https://github.com/artifacthub/hub/blob/c73dafa519020415927665e14fb6eac1066120eb/.github/workflows/ci.yml#L46-L57) 🙂

Hope this helps!
